joomla/joomla-cms
# Language definitions missing in formattedtext.php

When you update Joomla, but the log folder is not writable, appears the error:
Cannot write to log file.

The language definition is not present and so it is not translatable in other languages than english.

Pull Request for Issue # .

### Summary of Changes
Added the language definition in formattedtext.php and in en-GB.lib_joomla.ini

### Testing Instructions
1) Set the logs folder unwritable

2) Enable the laguage debug in global configuration

3) Go in the Optons of Joomla Update component and set the parameter Update Channel to Testing, so Joomla will find an update to do. Click on "Install the Update" button and look the error. 
You will see that the string "Cannot write to log file." ha not language definition.

### Documentation Changes Required
